File: Sum of n Numbers/Sum of n Numbers.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Tue Oct  6 18:22:24 2020

@author: ravi
"""

n = int(input("Please enter number ==> "))

res = 0

for i in range(n+1):
    res += i
print("Sum is ",res)

File: Sum of n Numbers/Sum of n Numbers.cpp
#include<iostream>
using namespace std;

int main(){
    int n;
    cout<<"Please enter Number ==> ";
    cin>>n;

    int res = 0;
    for(int i=0;i<=n;i++){
        res += i;
    }

    cout<<"Sum is ==> "<<res;
    return 0;
}